docs/trino-connector/catalog-hive.md

```diff
@@ -55,44 +76,68 @@ Support for the following alter table operations:
-  format = 'TEXTFILE'
+  format = 'TEXTFILE',
+  KEY = 'VALUE',
+  ...      
 );
 ```
 
-| Property      | Description                             | Default Value                                              | Required | Since Version |
-|---------------|-----------------------------------------|------------------------------------------------------------|----------|---------------|
-| format        | Hive storage format for the table       | TEXTFILE                                                   | No       | 0.2.0         |
-| total_size    | Total size of the table                 | (none)                                                     | No       | 0.2.0         |
-| num_files     | Number of files                         | 0                                                          | No       | 0.2.0         |
-| external      | Indicate whether it's an external table | (none)                                                     | No       | 0.2.0         |
-| location      | HDFS location for table storage         | (none)                                                     | No       | 0.2.0         |
-| table_type    | The type of Hive table                  | (none)                                                     | No       | 0.2.0         |
-| input_format  | The input format class for the table    | org.apache.hadoop.mapred.TextInputFormat                   | No       | 0.2.0         |
-| output_format | The output format class for the table   | org.apache.hadoop.hive.ql.io.HiveIgnoreKeyTextOutputFormat | No       | 0.2.0         |
-| serde_lib     | The serde library class for the table   | org.apache.hadoop.hive.serde2.lazy.LazySimpleSerDe         | No       | 0.2.0         |
-| serde_name    | Name of the serde                       | table name by default                                      | No       | 0.2.0         |
-
-## Schema properties
-
-| Property | Description                     | Default Value | Required | Since Version |
-|----------|---------------------------------|---------------|----------|---------------|
-| location | HDFS location for table storage | (none)        | No       | 0.2.0         |
+The following tables are the properties supported by the Hive table:
+
+| Property       | Description                             | Default Value                                              | Required | Since Version |
+|----------------|-----------------------------------------|------------------------------------------------------------|----------|---------------|
+| format         | Hive storage format for the table       | TEXTFILE                                                   | No       | 0.2.0         |
+| total_size     | Total size of the table                 | (none)                                                     | No       | 0.2.0         |
+| num_files      | Number of files                         | 0                                                          | No       | 0.2.0         |
```

Comment:
Is these attributes like `total_size` and `num_files` can be set by the user?

Reply:
I don't think so, the word `The following tables are the properties supported by the Hive table` does not specify whether users can set it.